src/Umbraco.Core/Models/Member.cs

```diff
@@ -552,9 +552,15 @@ public IMemberType ContentType
         private Attempt<T> WarnIfPropertyTypeNotFoundOnGet<T>(string propertyAlias, string propertyName, T defaultVal)
         {
             void DoLog(string logPropertyAlias, string logPropertyName)
-                => Current.Logger.Warn<Member>($"Trying to access the '{logPropertyName}' property on " + typeof(Member)
-                + $" but the {logPropertyAlias} property does not exist on the member type so a default value is returned. Ensure that you have a property type with alias: "
-                + logPropertyAlias + $" configured on your member type in order to use the '{logPropertyName}' property on the model correctly.");
+            {
+                //WB: TODO Check that we do not need to specify properties again if used later on again in message template
```

Comment:
Does this TODO still need a follow up?

Reply:
I need to test this out - will check & verify